apps/google-analytics-4/frontend/src/components/config-screen/api-access/ApiAccessPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import InstalledServiceAccountCard from './service-account/InstalledServiceAccountCard';
+import type { ServiceAccountKey, ServiceAccountKeyId } from '../../../types';
+import SetupServiceAccountCard from './service-account/SetupServiceAccountCard';
+import { Subheading, Paragraph, Stack } from '@contentful/f36-components';
+
+interface Props {
```

Comment:
Common Pattern Comment: Define your Component prop types as `Props`

Reply:
Not something like `ApiAccessPageProps`?

Not opposed, just a new convention for me

Reply:
I'm happy to go either way though I've always done the above. I'll try to give both sides of the argument that I see

Being verbose doesn't add anything here since these Prop definitions are always internal to the component and makes everything consistent across all Component prop types.
vs.
Being explicit/verbose could have value here (I don't have an example on hand but that doesn't mean it doesn't exist)

Reply:
I like your reasoning and appreciate you sharing. The possible value would be in exported interfaces, which is an unusual circumstance, but not unheard of.